src/screens/Home.js

```diff
@@ -1,9 +1,22 @@
 import { render } from '../shared/dom.js';
+import { MovieTile } from '../components';
+import ApiProvider from '../providers/ApiProvider.js';
 
 const template = `
-    <div id="home-screen">This is home screen</div>
+    <div>
+        <div id="home-screen">This is home screen</div>
+        <template id="home-movie-tiles"></template>
+    </div>
 `;
 
-export const HomeScreen = ({ renderOn, options }) => {
+export const HomeScreen = async ({ renderOn, options }) => {
     render({on: renderOn, html: template});
-};
+
+    const movies = await ApiProvider.search('Fast & Furious');
```

Comment:
Please add TODO Comment here that this is temporary for now :D

Reply:
Up

Reply:
Done